### DISCUSSION TITLE
New option for the calendar without year parameter or month

### DISCUSSION THREAD

**Discussion by walturbo** [joomla/joomla-cms#39242] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
For example, in a site of recipes could i have the need publish a module only in christmas period with christmas recipes from december to january every year. One for period of Halloween, one for Valentine's day etc.
Another example could be activating module with a promotion from 1 august to 15 august every year.
Actually with "year parameter" is necessary change date every year risking forget activate of module.

### Describe the solution you'd like
Have the possibility to omit the "year" or "month" field from the calendar settings.

### Additional context
This new feature can is utilized also for the articles, banner and other

**Comment by brianteeman** [comment]
Nice idea. However it would need more than just an option for the calendar without year parameter or month. All the queries that check for dates would also need to be changed.

**Comment by alikon** [comment]
let's move it to discussion
